src/System.IO.Pipelines.Networking.Tls/OpenSslConnectionContext.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO.Pipelines.Networking.Tls.Internal.OpenSsl;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+
+namespace System.IO.Pipelines.Networking.Tls
+{
+    public class OpenSslConnectionContext : ISecureContext
+    {
+        private const Interop.ContextOptions _contextOptions = Interop.ContextOptions.SSL_OP_NO_SSLv2 | Interop.ContextOptions.SSL_OP_NO_SSLv3;
```

Comment:
Again, we should default to letting the OS configuration decide, not defaulting to just picking what is best at the time of compilation.